### PR TITLE
Revise "softmax-regression_vn" - Phần 2

### LINKED ISSUE #911: Revise "softmax-regression_vn" - Phần 2
Phần này được dịch bởi:
* **NhatQuanBui**
* **lyphilong**

Nếu bạn đã dịch phần này, vui lòng bỏ qua việc revise.

### PR DESCRIPTION
**Pull request by quangnhat185**
## Pull Request Checklist

**Đánh dấu tick (sau khi "Create Pull Request") vào các mục dưới đây:**

* [x] Pull Request này tương ứng với issue nào? Trả lời số issue sau dấu `#` (không có dấu cách): Close #911 

* [x] Bản dịch này có bám sát **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)** không? Nếu một từ/cụm chưa có trong Bảng thuật ngữ, hãy tạo một Pull Request riêng đề xuất cách dịch từ/cụm đó.

* [x] Format của bản gốc có được giữ nguyên không?
    * Các phần in nghiêng, in đậm.
    * Tiêu đề (số lượng dấu `#` đầu dòng).
    * Bảng biểu, chú thích cho bảng (dòng phía trên mỗi bảng bắt đầu bằng dấu `:`).
    * Dịch các từ trong hình vẽ nếu cần. Các dòng có hình có dạng: `caption`.
    * Dịch các chú thích hình vẽ (thay các cụm `*dịch chú thích ảnh phía trên*` bằng bản dịch tiếng Việt).
    * Không dịch các phần code (nằm giữa hai dấu `````).
    * Copy các công thức toán từ bản gốc (các đoạn có `$`).
    * Giữ các dòng gán nhãn (bắt đầu với `:label:`).
    * Không tự thêm bớt các dòng trắng.
     
* [x] Trong một đoạn văn, mỗi câu văn đã được viết trong một dòng, giữa các dòng không có dòng trắng. Markdown sẽ coi những dòng liền nhau không có dòng trắng là một đoạn văn. Việc này giúp công đoạn review được thuận tiện hơn.

* [x] Tất cả chữ trong các hình vẽ đã được dịch. Nếu chưa, vui lòng xem thêm **[hướng dẫn hoặc yêu cầu dịch hình](https://github.com/aivivn/d2l-vn/blob/master/img/README.md)**.

* [x] Điền tên của bạn vào mục "Những người thực hiện" ở cuối file.

* [ ] **Sau khi được approve**, điền tên những reviewers mà bạn thấy có nhiều đóng góp cho Pull Request này. Xem danh sách Contributors **[tại đây](https://github.com/aivivn/d2l-vn/blob/master/docs/contributors_info.md)**.

## Với Reviewer

1. Chỉ review nếu Pull Request này đã sẵn sàng cho việc review.

2. Đã đọc kỹ hướng dẫn đóng góp và xem qua **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)**.

3. **Bạn có thể đề xuất thay đổi cách dịch mỗi dòng trực tiếp như trong hình dưới đây:**
img

### CODE REVIEW

File: chapter_linear-networks/softmax-regression_vn.md [hunk 1]
```diff
@@ -733,12 +733,15 @@ với dấu `@` ở đầu. Ví dụ: @aivivn.
...
 * Lê Khắc Hồng Phúc
+* Nguyễn Lê Quang Nhật
 
 <!-- Phần 3 -->
 *Bùi Nhật Quân
+* Nguyễn Lê Quang Nhật
 
 <!-- Phần 4 -->
 * Lý Phi Long
+* Nguyễn Lê Quang Nhật
```
Comment (tiepvupsu) [inline chapter_linear-networks/softmax-regression_vn.md]: Không sợ mọi người quên đâu em :D. Rồi sau cũng phải xóa các tên lặp đi.

Comment (quangnhat185) [reply chapter_linear-networks/softmax-regression_vn.md]: > Không sợ mọi người quên đâu em :D. Rồi sau cũng phải xóa các tên lặp đi.

Em chỉ tuân theo format của 1 PR thôi mà :(((

File: chapter_linear-networks/softmax-regression_vn.md [hunk 2]
```diff
@@ -215,7 +215,7 @@ These violate basic axioms of probability presented in :numref:`sec_prob`
 
 Bạn có thể muốn đề xuất rằng ta có thể lấy trực tiếp logit $o$ làm đầu ra để tiến hành dự đoán.
```
Comment (lkhphuc) [inline chapter_linear-networks/softmax-regression_vn.md]:
```suggestion
Bạn có thể muốn đề xuất rằng ta lấy trực tiếp logit $o$ làm đầu ra mong muốn.
```

File: chapter_linear-networks/softmax-regression_vn.md [hunk 3]
```diff
@@ -215,7 +215,7 @@ These violate basic axioms of probability presented in :numref:`sec_prob`
...
 Tuy nhiên, tại đây ta có một vài vấn đề khi lấy kết quả trực tiếp của tầng tuyến tính như một kết quả cho xác suất.
```
Comment (lkhphuc) [inline chapter_linear-networks/softmax-regression_vn.md]:
```suggestion
Tuy nhiên, sẽ có vấn đề khi coi kết qủa trả về trực tiếp từ tầng tuyến tính như là các giá trị xác suất.
```

File: chapter_linear-networks/softmax-regression_vn.md [hunk 4]
```diff
@@ -215,7 +215,7 @@ These violate basic axioms of probability presented in :numref:`sec_prob`
...
-Bởi vì, không có bất cứ điều kiện nào để ràng buộc tổng của những con số này bằng $1$.
+Lý do là không có bất cứ điều kiện nào để ràng buộc tổng của những con số này bằng $1$.
 Hơn nữa, tùy thuộc vào đầu vào mà ta có thể nhận được giá trị âm.
 Các điều kể trên đã vi phạm vào các tiên đề cơ bản của xác xuất đã được nhắc đến trong :numref:`sec_prob`
```
Comment (lkhphuc) [inline chapter_linear-networks/softmax-regression_vn.md]:
```suggestion
Các lý do trên khiến kết quả của tầng tuyến tính vi phạm vào các tiên đề cơ bản của xác xuất đã được nhắc đến trong :numref:`sec_prob`.
```

File: chapter_linear-networks/softmax-regression_vn.md (hunk 1 above)
Comment (duythanhvn) [reply chapter_linear-networks/softmax-regression_vn.md]: Thương thương :)))
